code/headers/mq_sensor.hpp

```diff
@@ -46,69 +48,49 @@ namespace r2d2::gas_detection {
          * divides it with the ro_clean_air_factor.
          * @return
          */
+        void calibrate() {
+            int val = 0;
```

Comment:
same here. What is val? Isn't there a variable name which makes it easier to understand?